front/lib/api/assistant/actions/retrieval.ts

```diff
@@ -445,7 +445,8 @@ export async function renderRetrievalActionByModelId(
 
 export function retrievalMetaPrompt() {
   return (
-    "To cite retrieved documents from data sources use the markdown directive :cite[REFERENCE]" +
+    "Focus on retrieved data and be factual." +
```

Comment:
So we still change this for everyone, right?

Reply:
Yeah I think that's fine.